bindings/rust/wasmedge-sys/examples/async_host_func2.rs

```diff
@@ -0,0 +1,46 @@
+use wasmedge_sys::{Executor, FuncType, Function, Store};
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    // create a FuncType
+    let func_ty = FuncType::create(vec![], vec![])?;
+    let mut store = Store::create().expect("123");
+    dbg!(&store.async_state.current_poll_cx.get());
+    // println!("{:?}", store.async_cx().unwrap());
```

Comment:
Redundant line.

Reply:
This is still a draft, and all these will be removed later.

Reply:
Ah, sorry, not noticed this is still a draft.